server/plugin/command.go

```diff
@@ -320,14 +333,25 @@ func (p *Plugin) handleSubscribesAdd(_ *plugin.Context, args *model.CommandArgs,
 
 	ctx := context.Background()
 	githubClient := p.githubConnectUser(ctx, userInfo)
-
+	user, _ := p.API.GetUser(args.UserId)
 	owner, repo := parseOwnerAndRepo(parameters[0], p.getBaseURL())
+	previouslySubscribed, previousSubscribedEvents, appErr := p.getSubscribedEvent(args.ChannelId, owner, repo)
```

Comment:
`appErr` is an `error`, not an `*model.AppError` and hence should be named `err`